includes/admin/tools/data/class-give-tools-reset-stats.php

```diff
@@ -133,9 +133,9 @@ public function get_data() {
 							WHERE $wpdb->terms.term_id IN
 							(SELECT $wpdb->term_taxonomy.term_id
 							FROM $wpdb->term_taxonomy
-							WHERE $wpdb->term_taxonomy.taxonomy LIKE 'give_forms_category'
-							OR $wpdb->term_taxonomy.taxonomy LIKE 'give_forms_tag')";
-						$sql[] = "DELETE FROM $wpdb->term_taxonomy WHERE $wpdb->term_taxonomy.taxonomy LIKE 'give_forms_category' OR $wpdb->term_taxonomy.taxonomy LIKE 'give_forms_tag'";
+							WHERE $wpdb->term_taxonomy.taxonomy = 'give_forms_category'
```

Comment:
@Sidsector9 use `prepare` function to make query safe

Reply:
@ravinderk Changes made.